test/cmd/debug.sh

```diff
@@ -525,3 +526,160 @@ run_kubectl_debug_netadmin_node_tests() {
   set +o nounset
   set +o errexit
 }
+
+run_kubectl_debug_sysadmin_tests() {
```

Comment:
Could you please check that why the integration test is failing?

Reply:
I'm not sure there is any strict policy for the integration tests and I'd prefer having these tests unless there is a very strong reasoning. Did you test when you modify this pod yaml as privileged?;

```yaml
apiVersion: v1
kind: Pod
metadata:
  labels:
    run: target
  name: target
spec:
  securityContext:
    seccompProfile:
      type: Localhost
      localhostProfile: dummy.json
  containers:
  - image: busybox
    name: target
    command: ["/bin/sh", "-c", "sleep 100"]
    securityContext:
        capabilities:
          add:
          - "ALL"
```

Reply:
you can test this also on your local with this command;

```sh
$ make test-integration WHAT=debug_sysadmin
```

Reply:
hmm, it is unfortunate because this integration test would be quite useful. But I think, it is better to skip test, since it is not reasonable to set all cluster as privileged.

Reply:
What about only adding the tests about simply validations, etc. that don't require system level privileges. And of course, with a note why we don't implement all the tests for sysadmin profile?

Reply:
Yes, let's skip the integration tests for sysadmin profile.

Reply:
After thinking on this, I think we can skip the integration tests @eiffel-fl. We have a good coverage of unit tests which should be sufficient for this case.